### PR TITLE
To Deploy kubernetes pods of OracleDatabase only on nodes in a particular nodepool having label "pool"

### PR DESCRIPTION
**Pull request by Mahi0911**
To Deploy Kubernetes pods of OracleDatabase only on nodes in a particular NodePool having label "pool" , Added label "pool" under nodeSelector in OracleDatabase/SingleInstance/helm-charts/oracle-db/templates/deployment.yaml

### CODE REVIEW

**Review by yunus-qureshi — COMMENTED** [review]
@Mahi0911 can we generalise the label name here instead of assuming to be pool always

**Review by gvenzl — CHANGES_REQUESTED** [review]
As per @yunus-qureshi:
> @Mahi0911 can we generalise the label name here instead of assuming to be pool always?

File: OracleDatabase/SingleInstance/helm-charts/oracle-db/values.yaml [hunk 1]
```diff
@@ -46,3 +46,8 @@ imagePullPolicy: Always
 ## container registry login/password
 imagePullSecrets:
 
+## Deploy only on nodes having required labels .
+## Format label_name : label_value . eg pool : sidb
+## Leave empty if there is no such requirement
+nodeLables:
```
Comment (yunus-qureshi) [inline OracleDatabase/SingleInstance/helm-charts/oracle-db/values.yaml:52]: @Mahi0911 plz fix this typo. nodeLables should be nodeLabels. Update all references

**Review by yunus-qureshi — APPROVED** [review]
Looks good for merge

**Review by gvenzl — APPROVED** [review]

**Pull request merged**